### DISCUSSION TITLE
Alerts not being generated from Sysmon / Sigma Playbook

### DISCUSSION THREAD

**Discussion by t1ndustries** [Security-Onion-Solutions/securityonion#7695] [Unsupported Versions] [answered]
SO Version: 2.3.100 (Standalone)
Installed via ISO into VirtualBox Lab
All services running according to so-status
No errors on Grid

On a Win10 host, Sysmon logs are shipped from winlogbeat. I am able to see sysmon events being generated on the hunt interface. When trying out Sigma plays, I set active Play 424 whoami execution from the Community Sigma plays that come with the SO install. After running whoami through a variety of methods (cmd on host, powershell on host , and also exectuted from my Lab's 'attacker' C2) I can see the whoami.exe process_creation events showing up in Hunt, but they don't seem to be generating alerts. Checked to make sure that the ElastiAlert config was being updated, and I see the .yaml generated in ```/opt/so/rules/elastalert/playbook/<PlayID>.yaml``` After reviewing the play's .yaml I am unable to see any reason why the Play alert wouldn't be triggering.

Any help would be much appreciated!

**Comment by defensivedepth** [comment, marked as answer]
Next place to look is `/opt/so/log/elastalert/elastalert.log` - run whoami to generate the logs again and then grep / tail the elastalert log - 

Should see logs like this (and it will show you when it tries to generate an alert):

![image](https://user-images.githubusercontent.com/954732/161842927-2bbb67c3-7f88-4678-b8a4-b2c382192864.png)


Also copy the query in the Elastalert rule file to Hunt and confirm you get a hit.

**Reply by t1ndustries** [reply]
So after booting my lab setup back up, executing whoami on host, and then monitoring the elastalert.log, the logs indicated:

``` 2022-04-09 01:05:39,026     INFO apscheduler.executors.default Running job "Rule: Whoami Execution - e425270ab (trigger: interval[0:03:00], next run at: 2022-04-09 01:08:41 UTC)" (scheduled at 2022-04-09 01:05:39.025317+00:00)
2022-04-09 01:05:39,059     INFO           elastalert Queried rule Whoami Execution - e425270ab from 2022-04-09 00:55 UTC to 2022-04-09 01:05 UTC: 2 / 2 hits
2022-04-09 01:05:44,587     INFO           elastalert Ran Whoami Execution - e425270ab from 2022-04-09 00:55 UTC to 2022-04-09 01:05 UTC: 2 query hits (0 already seen), 2 matches, 2 alerts sent
2022-04-09 01:05:44,587     INFO           elastalert Whoami Execution - e425270ab range 600
```

I checked Alerts on SecOnion web panel, and the alerts are now generating!

![Screenshot_2022-04-08_20-34-27](https://user-images.githubusercontent.com/102918509/162551691-e7e2fd0e-343a-47d7-af87-01c2b503f429.png)

Unsure why it wasn't generating alerts before, I had even attempted to restart the lab setup and re-generate alerts once before... Unsure of how to check why the alerts were not triggering before.

**Reply by defensivedepth** [reply]
You would grep through the previous elastalert log files.
